### PR TITLE
adding analytics platform

### PR DESCRIPTION
**Pull request by peppasnet**

### CODE REVIEW

File: applications_sbx/analytics/application.yaml
Comment (dleonardi95) [inline applications_sbx/analytics/application.yaml:1]: I think this was a test yaml of the superset.yaml, please remove it.

File: applications_sbx/analytics/backend.yaml
Comment (dleonardi95) [inline applications_sbx/analytics/backend.yaml:1]: Path entry should point to ionos_sbx and to the right folder

path: ionos_dev/analytics/backend -> ionos_sbx/analyticsstack

File: applications_sbx/analytics/postgres-analytics.yaml
Comment (dleonardi95) [inline applications_sbx/analytics/postgres-analytics.yaml:1]: also here you should point to ionos_sbx and to the right folder and tyo the right resource, I see no postregs inside analytics folder.

File: applications_sbx/analytics/superset.yaml
Comment (dleonardi95) [inline applications_sbx/analytics/superset.yaml:1]: please point also here the resource under ionos_sbx

**Review by dleonardi95 — COMMENTED** [review]
Please take a look at these comments

File: ionos_sbx/analytics/superset/superset-sealed-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: bitnami.com/v1alpha1
```
Comment (bialask-daiteap) [inline ionos_sbx/analytics/superset/superset-sealed-secret.yaml:2]: Please put it into /templates subfolder

**Review by bialask-daiteap — APPROVED** [review]

File: applications_sbx/analytics/backend.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  namespace: argocd
+  labels:
+    purpose: analytics
+spec:
+  destination:
+    namespace: analytics
+    server: https://kubernetes.default.svc
+  project: default
+  source:
+    path: ionos_sbx/analytics/backend
```
Comment (dleonardi95) [inline applications_sbx/analytics/backend.yaml:14]: path: ionos_sbx/analytics/backend

this does not exist under ionos_sbx, please make the needed changes

**Pull request merged**